helm-chart/renku/values.yaml

```diff
@@ -892,8 +892,8 @@ notebooks:
   sessionIngress:
     host:
     tlsSecret:
+    className: "" # nginx
     annotations:
-      kubernetes.io/ingress.class: nginx
```

Comment:
issue: This is a breaking change. Make the default `nginx`, not `""`. And remove the `nginx` comment.

Reply:
Done